pandas/core/arrays/base.py

```diff
@@ -521,6 +521,92 @@ def argsort(
         result = nargsort(self, kind=kind, ascending=ascending, na_position="last")
         return result
 
+    def argmin(self):
+        """
+        Return the minimum argument indexer.
+
+        Returns
+        -------
+        scalar
+            Minimum argument indexer.
+
+        See Also
+        --------
+        ExtensionArray.argmax
+        """
+        if len(self) == 0:
+            raise ValueError("attempt to get argmin of an empty sequence")
+
+        return self.argsort()[0]
+
+    def min(self):
```

Comment:
What's the reason for splitting this into two methods `min` and `_min`?

Do we check whether `np.min` works on an extension array?